src/site/xdoc/checks/javadoc/javadocvariable.xml.template

```diff
@@ -35,7 +35,7 @@
         </macro>
         <p id="Example1-code">
           By default, this setting will report a violation if
-          there is no javadoc for any scope member.
+          there is no javadoc for a member with any access modifier.
```

Comment:
Please change all mention of `member` or `variable` to `field` in the docs to be consistent.